admin_guide/managing_projects.adoc

```diff
@@ -376,3 +379,51 @@ ifdef::openshift-enterprise[]
 # systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 ----
 endif::[]
+endif::openshift-enterprise,openshift-origin,openshift-dedicated[]
+
+ifdef::openshift-dedicated[]
+[[enforcing-different-linmits-on-users-projects]]
+== Enforcing Different Limits on Users' Projects
+
+If you are an {product-title} administrator and want to enforce different limits
+on the number of projects that your users can create, apply the appropriate
+label to the user.
+
+The admission controller must be configured for all possible values for the
+project limits. The default limit is `unlimited`.
+If you want to enforce different limits on the number of projects that your
+users can create, apply the appropriate label to the user.
+
+The admission controller must be configured for all possible values for the
+project limits. The default limit is `unlimited`.
+
+.Example Project Limit for Users
+
+----
+'openshift.io/project-limit':3
+'openshift.io/project-limit':20
+----
+
+[[managing-limits-and-quota-on-project-resources]]
+== Managing Limits and Quota on Project Resources
+
+There is a default project template available for any projects created by users
+on the cluster, which helps users from inadvertently consuming all of cluster
+and node resources. The project template is created and configured in the
+`dedicated-admin` namespace.
+
+There is an annotation on the project template that governs whether the template
+will be updated during upgrades or not:
+
+----
+openshift.io/overwrite-protect:true|false
+----
+
+{product-title} administrators can customize the default project template
+without getting changes overwritten by:
+
```

Comment:
@abhgupta What are the steps that should be included here? Thanks!

Reply:
They need to update the annotation value to "true" (yes, protect against overwrite). 
Its "false" by default (no, do not protect against overwrite).

`'openshift.io/overwrite-protect': true|false`